seaborn/categorical.py

```diff
@@ -3889,14 +3891,15 @@ def catplot(
         Categorical variables that will determine the faceting of the grid.
     {col_wrap}
     {stat_api_params}
-    {order_vars}
+    {order_vars} ``order`` is ignored if the ``share`` parameter for the
```

Comment:
This doesn't work, unfortunately. It renders as:

```
order, hue_order : lists of strings, optional
    Order to plot the categorical levels in, otherwise the levels are
    inferred from the data objects.         ``order`` is ignored if the ``share`` parameter for the
    categorical axis is ``False``.
```

Reply:
Although obsolete code now, this construct rendered ok in the result HTML output (there wasn't a gap there).